snap_http/api.py

```diff
@@ -220,11 +220,25 @@ def list() -> SnapdResponse:
 # Configuration: get and set snap options
 
 
-def get_conf(name: str) -> SnapdResponse:
-    """Get the configuration details for the snap `name`."""
-    return http.get(f"/snaps/{name}/conf")
+def get_conf(name: str, *, keys: Optional[List[str]] = None) -> SnapdResponse:
+    """Get the configuration details for the snap `name`.
+
+    :param name: the name of the snap.
+    :param keys: retrieve the configuration for these specific `keys`. Dotted
+        keys can be used to retrieve nested values.
+    """
+    query_params = {}
+    if keys:
+        query_params["keys"] = ",".join(keys)
```

Comment:
We'll probably run into issues because of how mutable default arguments behave in Python: https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments

Reply:
I don't see you mutating `keys`, and I would be _very_ vocal if I saw someone mutating an argument in my (mostly) pure code-base.

Reply:
But it's also not a big deal here, so I'll allow it.